.github/workflows/scorecard.yml

```
@@ -0,0 +1,46 @@
+name: OpenSSF Scorecard
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)